bugbug/bug_features.py

```diff
@@ -333,6 +345,7 @@ def transform(self, bugs):
         results = []
 
         reporter_experience_map = defaultdict(int)
+        author_ids = None if self.commit_map is None else get_author_ids()
```

Comment:
Do it the other way around, like for commit_map, it is more readable

```Python
author_ids = get_author_ids() if self.commit_map else None
```

Reply:
This is not working if commit_map is empty.
```
cm = {}
>>> 1 if cm else 0
0
```
I can do this instead
```
>>> 1 if cm is not None else 0
```

Reply:
When is commit_map empty?

Reply:
Ok so i mined small data(1000) using new template, and all those commits contains `bug_id = ""` so `get_commit_map` is returning `{} dictionary. `     
So you want to me to continue with this one? 
`author_ids = get_author_ids() if self.commit_map else None`

Reply:
Ah ok, yeah in general it should never be empty, unless you have wrong data in the commits.json DB (you might want to redownload it).